Editor/MenuActions/Export/Export.cs

```diff
@@ -5,13 +5,17 @@
 using UnityEngine.ProBuilder.Stl;
 using UnityEngine.ProBuilder;
 using Object = UnityEngine.Object;
+#if UNITY_2023_2_OR_NEWER
+using UnityEditor.Actions;
+using UnityEngine.UIElements;
+#endif
```

Comment:
These are not necessary :-)

Reply:
Fixed in [98f445b](https://github.com/Unity-Technologies/com.unity.probuilder/pull/468/commits/98f445b270e264883a6fd80ac90e35713021c8c5)